lib/lcov.rb

```diff
@@ -12,7 +12,7 @@ def generate_base_command_line(compiler, build_dir)
   end
 
   def generate_filter_command_line(build_dir)
-    "cd #{build_dir} && lcov -r ./lcov.output `pwd`/\\* -o ./lcov.output.filtered"
+    "cd #{build_dir} && lcov -r ./lcov.output `pwd`/\* -o ./lcov.output.filtered"
```

Comment:
This is emitting an error when running on Ubuntu 24, or a newer Ruby, or a newer lcov, I'm not sure.  Either way, both seem to work fine on the old Decent CI machines, and this works on the new one.  So should be fine.